driftdetect/driftdetector.py

```diff
@@ -0,0 +1,114 @@
+import json
+import logging
+from driftdetect.driftdetectortype import DriftDetectorType
+from marshmallow import Schema, fields, post_load, ValidationError
+
+
+logger = logging.getLogger(__name__)
+
+
+def _build_drift_insight(graph_result):
+    """
+    Build drift insight
+    :type BoltStatementResult
+    :param graph_result: Graph data returned by the validation_query
+    :return: Dictionary representing the addition data we have on the drift
+    """
+
+    data = {}
+    for k in graph_result.keys():
+        data[k] = graph_result[k]
+
+    return data
+
+
+class DriftDetector(object):
+    """
+    Class that represent a job to run against the graph. This is a series of
+    GraphStatement that will run sequentially
+    """
```

Comment:
Remove this inaccurate docstring.

Reply:
Should I create a new one instead or just remove it?